examples/compressible/skamarock_klemp_nonhydrostatic.py

```diff
@@ -69,13 +71,13 @@
     output = OutputParameters(
         dirname=dirname,
         dumpfreq=dumpfreq,
-        pddumpfreq=dumpfreq,
-        dumplist=['u'],
+        dump_nc=True,
+        dump_vtus=False
```

Comment:
I think I'd prefer for these changes not to be made here to avoid confusing things, so it'd be good to revert back to the previous outputting settings